<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni::ui::ProjectorSetupWidget</class>
 <widget class="QWidget" name="omni::ui::ProjectorSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="omni::ui::ProjectorGroupBox" name="groupBox">
     <property name="title">
      <string>Projector #1</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QFrame" name="frameSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::WrapLongRows</enum>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>1</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item row="8" column="0">
          <widget class="QLabel" name="lbLens">
           <property name="toolTip">
            <string>Throwratio</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/Projector_Throw_small.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="boxThrowRatio">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="lbFOV">
           <property name="toolTip">
            <string>Field of view</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/Projector_FOV_small.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="boxFOV">
           <property name="minimum">
            <double>20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>160.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="lbScreen">
           <property name="text">
            <string>Screen</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="cbScreen"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="lbSubScreen">
           <property name="text">
            <string>Subscreen</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSpinBox" name="boxSubScreen"/>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="lbWidth">
           <property name="toolTip">
            <string>Screen width</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/Screen_Width_small.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="boxWidth">
           <property name="minimum">
            <number>512</number>
           </property>
           <property name="maximum">
            <number>2048</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="lbHeight">
           <property name="toolTip">
            <string>Screen height</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/Screen_Height_small.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QSpinBox" name="boxHeight">
           <property name="minimum">
            <number>256</number>
           </property>
           <property name="maximum">
            <number>2048</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="value">
            <number>768</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omni::ui::ProjectorGroupBox</class>
   <extends>QGroupBox</extends>
   <header>omni/ui/ProjectorGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/ODM.qrc"/>
  <include location="../rc/style.qrc"/>
  <include location="style.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>groupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>toggleSettings(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxFOV</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>fovChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxThrowRatio</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>throwRatioChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScreen</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>screenChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxSubScreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>subScreenChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxWidth</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>resolutionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxHeight</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorSetupWidget</receiver>
   <slot>resolutionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleSettings(bool)</slot>
  <slot>toggleScreenSettings(bool)</slot>
  <slot>fovChanged(double)</slot>
  <slot>throwRatioChanged(double)</slot>
  <slot>screenChanged(int)</slot>
  <slot>subScreenChanged(int)</slot>
  <slot>resolutionChanged()</slot>
 </slots>
</ui>
